src/crimson/os/seastore/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ add_library(crimson-seastore
   onode_manager/simple-fltree/onode_delta.cc
   onode_manager/simple-fltree/onode_node.cc
   seastore.cc
+  ../../../test/crimson/seastore/test_block.cc
```

Comment:
why? i don't think it's a part of seastore.

Reply:
cache.cc needs to be able to generate a TestBlock object for unittests.  At some point I may make it more dynamic, but this is simpler for now.